controllers/utils/handle_messages.go

```diff
@@ -70,6 +70,7 @@ func HandleVGCErrorMessage(logger logr.Logger, client client.Client, vgc *volume
 		if uErr := createNamespacedObjectErrorEvent(logger, client, vgc, errorMessage, reason); uErr != nil {
 			return uErr
 		}
+		return err
```

Comment:
Instead of return err inside the if - return err at the end of the function (instead of nil)
That apply to all functions in this file

Reply:
done